netsetup/__main__.py

```diff
@@ -18,8 +18,8 @@
 
 import daemon
 from .desktop_bus import DbusService
-from .wpantun import Wpantun
-from .ble import Ble
+from .wpantun import WpanService
```

Comment:
I would change this to WpanProxy or WpanClient because it does not seem a service that we are implementing. Like the DbusService.
Does it make sense?

Reply:
I also would change the commit message because are no longer changing just local variables

Reply:
Ok